doc/ref_cert/README.md

```diff
@@ -144,10 +144,10 @@ CNTT requirements about verification, validation, compliance, and conformance:
   for third-party conformance review
 
 Here are a couple of publicly available playbooks :
-- [Xtesting samples](https://git.opnfv.org/functest-xtesting/plain/ansible/site.yml?h=stable/leguer)
-- [OpenStack verification](https://git.opnfv.org/functest/plain/ansible/site.yml?h=stable/leguer)
-- [CNTT RC1](https://git.opnfv.org/functest/plain/ansible/site.cntt.yml?h=stable/hunter)
-- [Kubernetes verification](https://git.opnfv.org/functest-kubernetes/plain/ansible/site.yml?h=stable/leguer)
+- [Xtesting samples](https://git.opnfv.org/functest-xtesting/plain/ansible/site.yml?h=stable/wallaby)
+- [OpenStack verification](https://git.opnfv.org/functest/plain/ansible/site.yml?h=stable/wallaby)
```

Comment:
@collivier Since this is for Kali release, should the OpenStack release be "wallaby"?

Reply:
I considered both options.
Here it's about any OpenStack verification and then I selected the latest Functest stable release.
In any case, I would consider Xtesting wallaby is the latest answer here.

@pgoyal01 
I can switch to Jerma if it helps the reader. not a big deal. Waiting your answer.

Reply:
@collivier Thanks -- let us leave it as Wallaby.